### PR TITLE
Feature/improve filters tags

### LINKED ISSUE #773: Improve filters
More Filters options with dropdown.

Check loom video

### PR DESCRIPTION
**Pull request by adilrime17**
Had a discussion with @mghazanfar about test checks.
After running the yarn test and pressing a for a run all tests.
image
However, I have made some prop changes in the component. But when I go to the test file of that respective component, my editor shows some errors which I fixed.
This is discussed with @mghazanfar.

Feature is done by the way. Please review PR.
This PR Closes #773

### CODE REVIEW

File: src/common/components/entry-index-menu-dropdown/index.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  if (filter === 'created') {
+    dropDownItems = [
+      ...dropDownItems,
+      // for adding new menu items - example shown below
+      // {
+      //   label: <span>Now</span>,
+      //   active: tag === "right_now",
+      //   onClick: () => console.log('right_now clicked'),
+      // },
+    ]
```
Comment (mghazanfar) [inline src/common/components/entry-index-menu-dropdown/index.tsx:46]: @adilrime17 Can we delete this commented code if it is of no use?

File: src/common/components/entry-index-menu/_index.scss [hunk 2]
```diff
@@ -23,6 +23,34 @@
...
+      // margin-bottom: 8px;
+      padding: 4px 8px;
+      text-decoration: none;
+      transition: 0.5s;
+      font-size: 90%;
+  
+      // @media (min-width: $lg-break) {
+      //   font-size: $font-size-base;
+      //   padding: 4px 12px;
+      // }
```
Comment (mghazanfar) [inline src/common/components/entry-index-menu/_index.scss:39]: Same

File: src/common/components/entry-index-menu/_index.scss [hunk 3]
```diff
@@ -23,6 +23,34 @@
...
+  
+      @include tag;
+
+      svg {
+        // height: 14px;
+        width: 20px;
+        // margin-right: 4px;
+      }
```
Comment (mghazanfar) [inline src/common/components/entry-index-menu/_index.scss:47]: Same

**Review by mghazanfar — CHANGES_REQUESTED** [review]

File: src/common/components/entry-index-menu-dropdown/index.tsx (hunk 1 above)
Comment (adilrime17) [reply src/common/components/entry-index-menu-dropdown/index.tsx:46]: I have purposely kept this comment so in the future when we add values then we have an example in there and in this way we need not go through the code to understand the complete logic. But if you still want me to remove this then let me know I will remove and update PR as well.

**Review by feruzm — APPROVED** [review]
lgtm

**Review by mghazanfar — APPROVED** [review]

**Pull request merged**